var/www/html/tools/opcache/index.php

```diff
@@ -0,0 +1,744 @@
+<?php
```

Comment:
What's the origin and license for this file?

Should it be forked in this repo, or installed via the Dockerfile?